packages/flame_behaviors/lib/src/behaviors/propagating_collision_behavior.dart

```diff
@@ -106,6 +106,10 @@ class PropagatingCollisionBehavior<Parent extends EntityMixin>
   /// or a [Entity]. If it is neither, it will return [other].
```

Comment:
We should update the documentation to reflect the latest changes. For example, now it can return null even when the parent is of the right type but it hasn't been mounted.